src/CSS.cpp

```diff
@@ -1,9 +1,11 @@
 #include "CSS.h"
 #include "Common.h"
 
+#include <sass/context.h>
 #include <string>
 #include <array>
 #include <fstream>
+#include "sass.h"
```

Comment:
One of the includes is uneccessary (I'd prefer a single ```<sass.h>``` seperated from the stl includes)

Reply:
fixed